custom_components/google_home/api.py

```diff
@@ -309,6 +325,49 @@ async def reboot_google_device(self, device: GoogleHomeDevice) -> None:
                 device.name,
             )
 
+    async def get_or_set_do_not_disturb(
+        self, device: GoogleHomeDevice, enable: Optional[bool] = None
+    ) -> GoogleHomeDevice:
+        """Gets or sets the do not disturb setting on a Google Home device."""
+
+        data = None
+
+        if enable is None:
+            _LOGGER.debug(
+                "Getting Do Not Disturb setting from Google Home device %s",
+                device.name,
+            )
+        else:
+            data = {"notifications_enabled": not enable}
+            _LOGGER.debug(
+                "Setting Do Not Disturb setting to %s on Google Home device %s",
+                not enable,
+                device.name,
+            )
+
+        response = await self.post(
+            endpoint=API_ENDPOINT_DO_NOT_DISTURB, data=json.dumps(data), device=device
+        )
+        if response:
+            if "notifications_enabled" in response:
```

Comment:
Perhaps we could move `"notifications_enabled"` into as we do in other places constants? :)